dashboard/app/controllers/pd/professional_learning_landing_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Pd::ProfessionalLearningLandingController < ApplicationController
+  before_action :require_admin
```

Comment:
since this is temporary, maybe a comment to that effect.

Consider adding a unit test for access (admin can view, regular users can't)